actors/runtime/src/main/java/cloud/orbit/actors/runtime/ActorProfiler.java

```diff
@@ -44,7 +43,7 @@
 {
     private final HashMap<Object, ProfilerData> collectedData = new HashMap<>();
     private final ReadWriteLock rwlock = new ReentrantReadWriteLock();
-    private final KryoCloner kryoCloner = new KryoCloner();
+    private final KryoSerializer kryoCloner = new KryoSerializer();
```

Comment:
Should rename this field to be "serializer" so it describes the capability needed and not the specific implementation.